package.json

```diff
@@ -60,8 +60,11 @@
     "@types/marked": "^6.0.0",
     "@typescript-eslint/eslint-plugin": "^6.11.0",
     "@typescript-eslint/parser": "^6.11.0",
+    "ajv": "^6.12.6",
```

Comment:
why is this dependency declared here?

Reply:
there were some conflicts with yarn before the move to v4. maybe this can be removed now.